react-frontend/src/components/Header/MenuIcon/MenuIcon.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import './MenuIcon.css'
+import PropTypes from 'prop-types'
+
+const MenuIcon = ({ onClick }) => {
+  return (
+    <div className="menu-icon" onClick={onClick}>
```

Comment:
I would prefer to use custom icon. I wanted to make sure it look different from reusable component button. 
Here is how it looks like:
![image](https://github.com/YaroslavKSE/AuctionWeb/assets/94047397/321ddd39-97a4-461c-b3cc-5e5084be1995)

Reply:
But it has nothing to do with the icon. I suggest replacing parent element, so your markup will be something like that in the result:
```jsx
<button className="menu-icon" onClick={onClick}>
  <div className="menu-icon__line"></div>
   <div className="menu-icon__line"></div>
  <div className="menu-icon__line"></div>
</buton>
```

All you need to do is just use semantically correct element instead of `div`

Reply:
I got your point, I am going to refactor it.

Reply:
Refactored in [d4b0b3f](https://github.com/YaroslavKSE/AuctionWeb/pull/2/commits/d4b0b3fda1ec969fe3c167b604e5507f8fce6ee7)

Reply:
But you have not updated the `button` tag in line 7

Reply:
Sorry I missed that. Refactored in [9667654](https://github.com/YaroslavKSE/AuctionWeb/pull/2/commits/9667654d41d44244545551d6fbdb5bedb5025c79)